src/jspdf.js

```diff
@@ -563,14 +563,14 @@ var jsPDF = (function (global) {
     };
 
     /**
-     * Adds a properties to the PDF document
+     * Adds a properties to the PDF document.
      *
      * @param {Object} A property_name-to-property_value object structure.
      * @function
      * @instance
      * @returns {jsPDF}
      * @memberOf jsPDF
-     * @name setProperties
+     * @name setDocumentProperties
      */
     var setDocumentProperties = API.__private__.setDocumentProperties = API.setProperties = function (properties) {
```

Comment:
```suggestion
    var setDocumentProperties = API.__private__.setDocumentProperties = API.setProperties = API.setDocumentProperties = function (properties) {
```

Well better to add this additional methodname and keep "setProperties" for backwards compatibility

Reply:
Hi @arasabbasi , thanks for answering! :-) I totally agree with you. I'm going to do it right now ...